schedule/yam/agama_generate_dud.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: agama_generate_dud
+description: >
+  Generate DUD file for later Agama unattended installation.
```

Comment:
could you please rename all that is necessary to call this scenario `_unattended` starting by the test suite name? (not referring to this file in particular, what make sense according to other examples).

Reply:
I will wait for this comment reply to do further changes on this comment
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22516#discussion_r2179308983